css/style.css

```diff
@@ -0,0 +1,144 @@
+*{
+    
+}
+body{
+    background-color: #eee;
+    font-family: 'Lato', sans-serif
+}
+.header{
+    display: flex;
+    padding-top: 25px;
+    background-color: #eee;
+    
+}
+.headerlink{
+    padding-left: 500px;
+    display: flex;
+    gap: 150px;
+}
+a{
+    color: black;
+    text-decoration: none;
+}
+.original{
+    font-size: 20px;
+    display: flex;
+    gap: 30px;
+}
+
+img{
+    width: 400px;
+}
+
+.hero{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 200px;
+    margin-top: 50px;
+    
+}
+.hero > h2 {
+    margin-bottom: 20px;
+    word-wrap: break-word;
+  }
```

Comment:
please check for double lines in code

Reply:
- [ ] Please make sure when you write your comment the first letter is capitalized 🙃 😉
- [ ] Also be more clear in your comment when you suggest the change 😁 😁, and use a couple of emojis and/or gifs in your review, they make the code review more human-friendly 👽🤖.